dist/tools/functions.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# run arguments as command, only print output when command errors
+quiet() {
+    OUT="$("$@" 2>&1)"
+    RES=$?
+    [ -n "$OUT" ] && [ $RES -ne 0 -o "$QUIET" = "0" ] && echo "$OUT"
+    return $RES
+}
+
+echo $QUIET
```

Comment:
do we need to print that? In the logs it looks like a random "1" popping out of nowhere.